### ISSUE TITLE
WMAP Targets -t Crashes

### ISSUE THREAD

**Issue by smcintyre-r7** [rapid7/metasploit-framework#17718]
I get a stack trace when trying to define a target site using `wmap_targets -t`. From the help output, I'm pretty sure it's supposed to be a URL.

```
msf6 exploit(multi/http/lucee_scheduled_job) > wmap_targets -h
[*] Usage: wmap_targets [options]
	-h 		Display this help text
	-t [urls]	Define target sites (vhost1,url[space]vhost2,url) 
	-d [ids]	Define target sites (id1, id2, id3 ...)
	-c 		Clean target sites list
	-l  		List all target sites

msf6 exploit(multi/http/lucee_scheduled_job) > wmap_targets -t http://metasploit.com
[-] Error while running command wmap_targets: PG::InvalidTextRepresentation: ERROR:  invalid input syntax for type inet: "metasploit.com"
CONTEXT:  unnamed portal parameter $2 = '...'


Call stack:
/home/smcintyre/.rvm/gems/ruby-3.0.4@metasploit-framework/gems/activerecord-7.0.4.2/lib/active_record/connection_adapters/postgresql_adapter.rb:783:in `exec_prepared'
/home/smcintyre/.rvm/gems/ruby-3.0.4@metasploit-framework/gems/activerecord-7.0.4.2/lib/active_record/connection_adapters/postgresql_adapter.rb:783:in `block (2 levels) in exec_cache'
/home/smcintyre/.rvm/gems/ruby-3.0.4@metasploit-framework/gems/activesupport-7.0.4.2/lib/active_support/concurrency/share_lock.rb:187:in `yield_shares'

...
```

**Comment by dwelch-r7** [comment]
@scythe000 I've got a PR up here that might fix your issue if you want to give it a go https://github.com/rapid7/metasploit-framework/pull/17747

**Comment by scythe000** [comment]
> @scythe000 I've got a PR up here that might fix your issue if you want to give it a go #17747

That works! Thank you!
